apps/console/components/pipeline/__tests__/headers.test.tsx

```diff
@@ -0,0 +1,114 @@
+import {
+  cleanup,
+  fireEvent,
+  render,
+  setup,
+} from "$fresh-testing-library/components.ts";
+import { afterEach, beforeAll, describe, it } from "$std/testing/bdd.ts";
+import { assert, assertEquals, assertExists } from "$std/testing/asserts.ts";
+import { Headers, HeadersType } from "../headers.tsx";
+
+describe("components/pipeline/headers.tsx", () => {
+  beforeAll(setup);
+  afterEach(cleanup);
+
+  const defaultProps: HeadersType = {
+    name: "testHeaders",
+    data: { testHeaders: { "Content-Type": "application/json" } },
+    errors: {},
+  };
+
+  /**
+   * TODO: uncomment those tests when id + htmlfor issue with headers is fixed
```

Comment:
Please remove these comments. Add the code once it is ready.

Reply:
Sorry about that, removed!